ompi/datatype/ompi_datatype.h

```diff
@@ -73,6 +73,7 @@ struct ompi_datatype_t {
 
     void*              args;                     /**< Data description for the user */
     void*              packed_description;       /**< Packed description of the datatype */
+    uint64_t           pml_data;                 /**< PML-specific information */
```

Comment:
This addition only affects the OMPI datatypes, everything below (and the real datatype engine is down in OPAL) remains unaffected.